routes/boxForm.js

```diff
@@ -1,12 +1,5 @@
 const router = require('express-promise-router')();
-const { db } = require('../config');
-
-const SQLQueries = {
-  CreateAnchorBox:
-    'INSERT INTO "Anchor_Box"(box_id, approved, message, zip_code, picture, general_location, date, launched_organically, additional_comments) VALUES($1, $2, $3, $4, $5, $6, $7, $8, $9)',
-  FindBoxId: 'SELECT box_id FROM "Anchor_Box" WHERE box_id = $1',
-  Return: 'RETURNING *',
-};
+const { findBoxId, createAnchorBox } = require('../services/boxFormService');
```

Comment:
🚫 **[eslint]** <[import/extensions](https://github.com/import-js/eslint-plugin-import/blob/v2.25.2/docs/rules/extensions.md)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Missing file extension for "../services/boxFormService"